Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -21,6 +23,8 @@
 static const int CACHE_VERSION = 13;  // Last changed in PR #3261
 static const int DATASTREAM_VERSION = QDataStream::Qt_5_5;
 
+static inline u64 GetTitleID(const QString&);
```

Comment:
I pointed this out before, but why is the definition of the function not just placed here instead of declaring a prototype for it? As far as I can tell, the definition has no dependency on anything declared or defined before it.

It should also be noted that this prototype is incorrect. GetTitleID itself is not currently static/internally linked (it should be), but this prototype is indicating that it is. `inline` is also unnecessary here it only ever appears in one translation unit (since it's declared and defined in the same cpp file).